src/trace-server.ts

```diff
@@ -86,6 +86,7 @@ export class TraceServer {
             await context?.workspaceState.update(key, none);
             await this.start(context);
         }
+        return this.getUrl(this.getSettings());
```

Comment:
Shouldn't it be `this.getClientSettings()`? The URL preference is for the client to know where to connect to. 

So, if we return here the client URL, why does the client (i.e. `vscode-trace-extension`) read from the preference directly?

Or do I misunderstand something?

Reply:
Per our discussion we do not need to return the url.  Changes have been made to the other PR, and this change is no longer needed.